cumulusci/files/templates/project/mapping.yml

```diff
@@ -0,0 +1,47 @@
+Insert Accounts:
+  sf_object: Account
+  table: accounts
+  fields:
+    Id: sf_id
```

Comment:
We should leave this out so OID's aren't stored in the db.  Without this mapping it will use internal database id's instead of OID's for reference.